test/ToucanRegenBridge.js

```diff
@@ -4,8 +4,10 @@ const { ethers } = require("hardhat");
 
 const { deployBridge } = require("../lib/bridge");
 const { prepareToucanEnv } = require("../lib/toucan");
+const { id } = require("ethers/lib/utils");
```

Comment:
The presence of `lib` here looks wrong to me - seems to be assuming knowledge of `ethers` internals.

I would suggest

```javascript
const { utils } = require("ethers");
const { id } = utils;
```

or just drop the second line and use `utils.id`.